OAI_CONFIG_LIST_sample

```diff
@@ -7,14 +7,14 @@
     {
         "model": "gpt-4",
         "api_key": "<your Azure OpenAI API key here>",
-        "api_base": "<your Azure OpenAI API base here>",
+        "base_url": "<your Azure OpenAI API base here>",
         "api_type": "azure",
         "api_version": "2023-07-01-preview"
     },
     {
         "model": "gpt-3.5-turbo",
         "api_key": "<your Azure OpenAI API key here>",
-        "api_base": "<your Azure OpenAI API base here>",
+        "base_url": "<your Azure OpenAI API base here>",
         "api_type": "azure",
```

Comment:
Dont we need Deployment Name for Azure OpenAI Configs?

Reply:
Suggestion, these are ;ot of changes, should we have practice to divide in small chunks to avoid overlooking?